src/Tribe/Context.php

```diff
@@ -514,6 +514,11 @@ protected function query_var( array $query_vars, $default ) {
 		$value = $default;
 
 		global $wp_query;
+
+		if ( ! $wp_query instanceof \WP_Query ) {
+			return $value;
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: bc8558323a9480979ed3585c222bf853</sub>